x/cdp/simulation/operations.go

```diff
@@ -61,7 +61,7 @@ func SimulateMsgCdp(ak auth.AccountKeeper, k keeper.Keeper, pfk types.PricefeedK
 		}
 
 		randCollateralParam := collateralParams[r.Intn(len(collateralParams))]
-		randDebtAsset := randCollateralParam.DebtLimit[r.Intn(len(randCollateralParam.DebtLimit))]
+		randDebtAsset := randCollateralParam.DebtLimit
 		randDebtParam, _ := k.GetDebtParam(ctx, randDebtAsset.Denom)
```

Comment:
```suggestion
		debtParam, _ := k.GetDebtParam(ctx, randCollateralParam.DebtLimit.Denom)
```

Rename to `debtParam` since debt param is no longer random here. Can combine the statements to be more concise since we've dropped `r.Intn()`.